packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -308,12 +311,12 @@ def draw_path(self, gc, path, transform, rgbFace=None):
 
         transform += Affine2D().scale(1, -1).translate(0, self.height)
         self._path_helper(self.ctx, path, transform, figure_clip)
+        if gc.stroke:
+            self.ctx.stroke()
 
         if rgbFace is not None:
             self.ctx.fill()
             self.ctx.fillStyle = '#000000'
-        else:
-            self.ctx.stroke()
```

Comment:
I think the stroke should always be drawn after the fill, so move it down here.

Reply:
On it :)